src/main.cpp

```diff
@@ -100,11 +104,13 @@ int main(int argc, char *argv[])
     }
 
     std::vector<unsigned char> bytecode;
+    // TODO - Allow reading debug information from file.
+    DebugInfo debug(buf.str());
 
     // Pretty hacky way of checking whether the input file is compiled or not.
     if (name[name.length()-1] != 'x') {
-
```

Comment:
There seem to be a lot of changes to whitespace in this function that wasn't there in the last pull request. Don't make gratuitous changes to whitespace, as it obscures the real purpose of the commit.